lib/facter/config.rb

```diff
@@ -312,8 +312,7 @@ module Config
           'operatingsystemrelease',
           'osfamily',
           'physicalprocessorcount',
-          'processor.*',
-          'processor*',
+          'processor[0-9]+.*',
```

Comment:
Can we have another ticket to have an integration/acceptance tests that blocks legacy facts and matches the resulted fact keys?